pkg/operator/ceph/cluster/osd/labels.go

```diff
@@ -51,6 +51,12 @@ func (c *Cluster) getOSDLabels(osd OSDInfo, failureDomainValue string, portable
 	labels[OsdIdLabelKey] = stringID
 	labels[FailureDomainKey] = failureDomainValue
 	labels[portableKey] = strconv.FormatBool(portable)
+	labels["app.kubernetes.io/name"] = AppName
```

Comment:
How about a common helper method for adding labels? Then we can avoid the code duplication.